config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java

```diff
@@ -1550,19 +1550,9 @@ void distributor_operation_cancelling_config_controlled_by_properties() throws E
         assertTrue(resolveDistributorOperationCancellationConfig(2));
     }
 
-    private boolean resolveDistributorSymmetricReplicaSelectionConfig(Boolean flagValue) throws Exception {
-        return resolveDistributorConfig((props) -> {
-            if (flagValue != null) {
-                props.setSymmetricPutAndActivateReplicaSelection(flagValue);
-            }
-        }).symmetric_put_and_activate_replica_selection();
-    }
-
     @Test
     void distributor_symmetric_replica_selection_config_controlled_by_properties() throws Exception {
-        assertFalse(resolveDistributorSymmetricReplicaSelectionConfig(null)); // defaults to false
-        assertFalse(resolveDistributorSymmetricReplicaSelectionConfig(false));
-        assertTrue(resolveDistributorSymmetricReplicaSelectionConfig(true));
+        assertTrue(resolveDistributorConfig((props) -> {}).symmetric_put_and_activate_replica_selection());
```

Comment:
This test should probably either be removed or renamed to imply the config default is always `true`

Reply:
Yes, thanks, removed